test/API/README.md

```diff
@@ -0,0 +1,86 @@
+# HDF5 API Tests
+
+This directory contains several test applications that exercise [HDF5](https://github.com/HDFGroup/hdf5)'s
+public API and serve as regression tests for HDF5 [VOL Connectors](https://portal.hdfgroup.org/display/HDF5/Virtual+Object+Layer).
+
+## Build Process and options
+
+These HDF5 API tests are enabled and built by default, but can be disabled if desired.
```

Comment:
disabled by default?

Reply:
There's going to be some weirdness because the intention was that they would be enabled by default. However, since this is close to a release, they've been temporarily disabled by default.